Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -460,27 +498,41 @@ bool EmuCodeBlock::WriteToConstAddress(int accessSize, OpArg arg, u32 address, B
 	}
 }
 
+#include "disasm.h"
```

Comment:
Oops -- that was debugging code that I left in. I haven't done a cleanup pass to remove those + printfs (I'll get them all later on).